source/ControllableHero.hx

```diff
@@ -0,0 +1,54 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+using flixel.util.FlxSpriteUtil;
+
+// Part 2: Added a ControllableHero Class
+class ControllableHero extends FlxSprite
+{
+	// Movement variable
+	var speed = 10; // Couldn't figure out a better way to handle this?
+
+	// Implement Constructor
+	public function new()
+	{
+		// Invoke parent class constructor
+		super(x, y);
+		this.x = x;
+		this.y = y;
+
+		// Make a simple rectangle sprite
+		var myRectangle = new FlxSprite(x, y);
+		makeGraphic(30, 10, 0xFFFFA500, true); // FlxColor Orange
```

Comment:
The 
```
var myRectangle = new FlxSprite(x, y);
```
Isn't impacting the game in a useful way.
Remember that `FlxSprite` class (and everything that inherits from it) is like a "character" or "visible, moveable object" in our game world - because this class inherits from `FlxSprite`, it's already going to have all of the methods and fields of its parent. It has the ability to render graphics and get drawn to the screen.

There's nothing that stops you from creating a new FlxSprite object here, but I don't think it ends up making much sense. Our "character" is the `ControllableHero` class, and your call to `makeGraphic` is what sets up its lovely orange rectangle. 

The `myRectangle` that you create here is a newly made FlxSprite, with all of the same underlying capabilites and fields as this class. If it were added to the playstate somehow, then you'd see it's little graphic on the screen too - you can think of each FlxSprite as its own separate entity, so it *usually* doesn't make sense for one thing in the game to create another.

The `makeGraphic` call is good, but you don't have to hard-code that `0xFFFFA500` value into the method call - instead you can just put in `FlxColor.ORANGE` and retrieve the value that FlxColor provides statically for anyone to use. The code would end up looking like
```
makeGraphic(30, 10, FlxColor.ORANGE, true);
```
(and you'd have this import added to the top of your file)
```
import flixel.util.FlxColor;
```
Using named variables like this definitely improves readability of the code without you having to write extra comments to explain what's going on. It's nice that you have the comment, since the hard-coded value would be much harder to interpret otherwise, but it's even better to not need it in the first place.